README.md

```diff
@@ -0,0 +1,17 @@
+# Akka Platform Operator Charts
+
+This repository contains Helm charts for the [Akka Platform](https://developer.lightbend.com/docs/akka-platform-guide/).
+
+## Usage
+
+Add this repository using the following commands:
+
+```
+helm repo add akka-operator-helm https://lightbend.github.io/akka-operator-helm/ 
+helm repo update
+```
+
+Install the operator (requires commercial license):
+```
+helm install akka-operator
```

Comment:
let's avoid too many instructions scattered at different places, link to the guide

Reply:
done